kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/base/BaseTest.java

```diff
@@ -24,6 +24,7 @@
 
 @Slf4j
 @DisplayNameGeneration(CamelCaseToSpacedDisplayNameGenerator.class)
+@SuppressWarnings("squid:S2187")
```

Comment:
?

Reply:
Sonar believes that a class with a name containing "Test" must contain tests. I'll see what I can do to make it look more elegant